commit 4fce2688c24bc758f96250413361aa97527275f6 (refs/changes/95/4795/1)
Author: Richard Kiene <richard.kiene@joyent.com>
Date:   2018-09-06T02:58:35+00:00 (1 year, 1 month ago)
    
    TRITON-755 Want ability to filter CMON discovery results by tag

diff --git a/package.json b/package.json
index 65e19a9..a16abbf 100644
--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "sdc-docker",
-  "version": "0.5.2",
+  "version": "0.5.3",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
@@ -28,7 +28,7 @@
     "strsplit": "1.0.0",
     "tape": "^4.4.0",
     "trace-event": "1.2.0",
-    "triton-tags": "1.1.4",
+    "triton-tags": "1.2.0",
     "ufds": "1.2.0",
     "vasync": "2.1.0",
     "verror": "1.9.0",
